src/index.js

```diff
@@ -0,0 +1,28 @@
+import displayList from './modules/display.js';
+import { addList, removeList, editList } from './modules/operation.js';
+import './index.css';
+
+const allList = JSON.parse(localStorage.getItem('todo')) || [];
+const bkList = document.querySelector('.lists');
+displayList(bkList, allList);
+const edit = document.querySelector('.editTask');
+const arrow = document.querySelector('.arrow');
+arrow.addEventListener('click', () => {
+  addList(allList, edit.value);
+  window.location.reload();
+});
```

Comment:
- [ OPTIONAL] Why don't you use the `submit` event on the form so that your form can be submitted with juts the press of the `enter` key? What do you think 💭